src/pybind/mgr/diskprediction/agent/metrics/db_relay.py

```diff
@@ -0,0 +1,561 @@
+from __future__ import absolute_import
+
+import socket
+
+from . import MetricsAgent
+from ...common.cypher import CypherOP, NodeInfo
+from ...common.db import DB_API
+from ...models.metrics.dp import DB_Relay
+from ...models.relation.mgrdp import MGRDpCeph, MGRDpMon, MGRDpOsd, \
+    MGRDpMds, MGRDpPG, MGRDpDisk, MGRDpRBD
+
+
+def get_human_readable(size, precision=2):

```

Comment:
The only difference between this and the function in sai_disk is that it's 1000 vs 1024 -- should create a shared function that takes an argument instead of having two almost identical functions.